test/jit/test_symbolic_shape_analysis.py

```diff
@@ -42,14 +42,34 @@ def foo(x, y):
         torch._C._jit_register_operator_shape_function(foo.graph.findNode("aten::mul"), broadcast.graph)
         inputs = list(foo.graph.inputs())
 
-        test_cases = [
-            ([1, 6, 5], [1, 7, 1, 5], "1, 7, 6, 5"),
-            ([1, 6, 5], [1, None, 1, 5], "1, *, 6, 5"),
-            ([None, None], [None, None, None], "*, *, *"),
-        ]
-
-        for inp0, inp1, result in test_cases:
+        def prop_shapes_on_graph(inp0, inp1):
             inputs[0].setType(inputs[0].type().with_sizes(inp0))
             inputs[1].setType(inputs[1].type().with_sizes(inp1))
             torch._C._jit_pass_propagate_shapes_on_graph(foo.graph)
-            FileCheck().check(result).run(foo.graph)
+
+        prop_shapes_on_graph([1, 6, 5], [1, 7, 1, 5])
+        FileCheck().check("1, 7, 6, 5").run(foo.graph)
+
+        prop_shapes_on_graph([None, None], [None, None, None])
```

Comment:
Does `None` mean symbolic shape? But then there is `new_sybolic_shape_symbol` used later. Adding a comment would help.